code/modules/research/designs/bot_upgrades.dm

```diff
@@ -0,0 +1,43 @@
+///////////////////
+// RESEARCH ITEM //
+///////////////////
+
+/datum/design/bot_upgrade
+	// Should not be acquirable
+
+/datum/design/bot_upgrade/boost
+	name = "Bot Upgrade: Boost"
+	desc = "The circuit board for giving a speed boost to bots"
+	id = "bot_upgrade_boost"
+	req_tech = list("programming" = 2, "engineering" = 2)
+	build_type = MECHFAB
+	materials = list(MAT_GLASS = 500)
```

Comment:
Is there a comparable item whose value I could use, as I'm unsure how much metal exactly 1 sheet is